### PR TITLE
Added linkedList examples

### PR DESCRIPTION
**Pull request by AndraB90**
2 examples to illustrate linkedList properties

### CODE REVIEW

File: src/linked_list/Example1.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        colorsLinkedList.add(3, "black");
+
+        System.out.println(colorsLinkedList);
+
+        /**
+         * LinkedList is slower for getting a value by index, because it needs to browse every index until the specific value
+         * O(n) linear time complexity
+         */
+        System.out.println(colorsLinkedList.get(4));
+
```
Comment (CorlyBoy) [inline src/linked_list/Example1.java:47]: extra space not needed

**Review by CorlyBoy — APPROVED** [review]
Looks good. You can resolve the comments if you want, or just merge the code to master. Approved!

File: src/linked_list/Example1.java (hunk 1 above)
Comment (AndraB90) [reply src/linked_list/Example1.java:47]: Resolved.

File: src/linked_list/Example2.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        lettersQueue.offer("C");
+        lettersQueue.offer("D");
+        lettersQueue.offer("F");
+        //lettersQueue.poll();
+        lettersQueue.add(4,"E");
+        lettersQueue.addFirst("0");
+        lettersQueue.addLast("1");
+        lettersQueue.removeFirst();
+        lettersQueue.removeLast();
+
```
[the remark this answers is not in the document]
Comment (AndraB90) [reply src/linked_list/Example2.java:32]: Resolved.

**Pull request merged**